app/src/main/java/io/github/jbellis/brokk/AnalyzerWrapper.java

```diff
@@ -95,6 +95,7 @@ public void onFilesChanged(EventBatch batch) {
                 logger.debug("Changes in git metadata directory ({}) detected", gitRepoRoot.resolve(".git"));
                 if (listener != null) {
                     listener.onRepoChange();
+                    listener.onTrackedFileChange(); // Tracked files can also change as a result, e.g. git add <files>
```

Comment:
I think this was the reason why it was already here before we removed it recently.